### DISCUSSION TITLE
xp-pen deco m

### DISCUSSION THREAD

**Discussion by fadri14** [OpenTabletDriver/OpenTabletDriver#2958] [General]
Hello, I have an xp-pen deco m graphics tablet and was wondering if it was supported as it's the only one in the deco series that doesn't appear in the list.

**Comment by jamesbt365** [comment]
If you have the tablet with you right now it can be supported, if it is not on the supported tablet list it is not currently supported, and we can't support tablets without users having them. We prefer to support tablets over in our [Discord](https://opentabletdriver.net/discord) server, as this is much faster. If you do not have a discord account you can create an issue using the configuration template.
